clang_3.8/Dockerfile

```diff
@@ -80,6 +81,7 @@ RUN dpkg --add-architecture i386 \
     && pip install -q --upgrade --no-cache-dir pip \
     && pip install -q --no-cache-dir conan conan-package-tools \
     && chown -R conan:1001 /opt/pyenv \
+    && find /opt/pyenv -iname __pycache__ -print0 | xargs -0 rm -rf \
```

Comment:
Find this a little bit tricky. Maybe it will benefit from a comment

Reply:
okay

Reply:
tried to use `PYTHONDONTWRITEBYTECODE` (https://docs.python.org/3/using/cmdline.html), but `__pycache__ ` is still created :( will add comment.